src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2615,6 +2615,13 @@ public static partial class GC
         /// </summary>
         /// <returns> A Dictionary of the Configuration Variables keyed on the name of the configuration with the values as the configuration values </returns> 
         public static System.Collections.Generic.IReadOnlyDictionary<string, object> GetConfigurationVariables() { throw null; }
+
+        public enum GCConfigurationType : short
```

Comment:
This shouldn't be in the public ref assembly.

Reply:
Also, the doc comments should not e included in the ref assembly. (You can delete the doc comments on `GetTotalPauseDuration` too while you are on it.)

Reply:
> Also, the doc comments should not e included in the ref assembly. (You can delete the doc comments on `GetTotalPauseDuration` too while you are on it.)

Where are they supposed to be? In all the individual implementations duplicated?

Reply:
There was a discussion about this https://github.com/dotnet/runtime/issues/44969#issuecomment-730684569 , but no solution was created for this problem. The doc comments are typically duplicated.

If you dislike the duplication, you can add it to coreclr impl only and omit it everywhere else. I believe that the doc comment porting tool takes the coreclr impl as the source of truth.